doc/nrf/protocols/matter/end_product/watchdog.rst

```diff
@@ -33,6 +33,22 @@ The feature is enabled by default for the release build type in all Matter sampl
 To set the timeout for the watchdog timer, configure the :kconfig:option:`CONFIG_NCS_SAMPLE_MATTER_WATCHDOG_TIMEOUT` with a value in milliseconds.
 By default, the timeout is set to 10 seconds.
 
+Matter watchdog pause mode
+**************************
+
+The Matter watchdog can be set to pause when the CPU is in idle (sleep) mode or when the CPU is halted by the debugger.
+This means the watchdog timer will stop counting down when the CPU is idle or halted.
+
+If pause mode is disabled, the watchdog must be fed within the specified time window, regardless of the CPU state, otherwise the device will reboot.
```

Comment:
```suggestion
If pause mode is disabled, the watchdog must be fed within a specified time window, regardless of the CPU state, otherwise the device will reboot.
```